libvdrtools/indy-api-types/Cargo.toml

```diff
@@ -13,15 +13,15 @@ rust-base58 = ["bs58"]
 
 [dependencies]
 failure = "0.1.8"
-futures = { version = "0.3", default-features = false }
+futures = { version = "0.3", default-features = false, features = ["std"] }
 log = { version = "0.4.17", features = ["std"] }
 libc = "0.2.114"
 openssl = {version = "0.10", optional = true}
 bs58 = {version = "0.4.0", optional = true}
 serde = "1.0.99"
 serde_json = "1.0.40"
 serde_derive = "1.0.99"
-sqlx = { version = "0.5.8", git = "https://github.com/jovfer/sqlx", branch = "feature/json_no_preserve_order_v5", features = [ "sqlite", "json_no_preserve_order" ], optional = true }
+sqlx = { version = "0.5.8", git = "https://github.com/jovfer/sqlx", branch = "feature/json_no_preserve_order_v5", features = [ "sqlite", "json_no_preserve_order", "runtime-tokio-rustls" ], optional = true }
```

Comment:
seems unrelated to pr, why the addition of `runtime-tokio-rustls`?

Reply:
On the `main` branch, `indy-api-types` and some of its dependents fail to build by themselves because

> one of the features ['runtime-actix-native-tls', 'runtime-async-std-native-tls', 'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', 'runtime-tokio-rustls'] must be enabled

The reason why `libvdrtools` or `aries_vcx` don't fail to build is because they add one of these features higher up the dependency chain.